fvtest/porttest/si.cpp

```diff
@@ -164,6 +164,13 @@ TEST(PortSysinfoTest, sysinfo_test0)
 		outputErrorMessage(PORTTEST_ERROR_ARGS, "  Executable name test failed: (rc = %d).\n", rc);
 		goto done;
 	}
+	
+	#if defined(OMR_OS_WINDOWS)
+	for (char* cursor = argv0; *cursor != '\0'; ++cursor) {
+		if (*cursor == '/') {
+			*cursor = '\\';
+		}
+	}
```

Comment:
I think you're missing the `#endif` here. The ifdef should also begin a column 0, so:

```
#if defined(OMR_OS_WINDOWS)
	for (char* cursor = argv0; *cursor != '\0'; ++cursor) {
		if (*cursor == '/') {
			*cursor = '\\';
		}
	}
#endif /* defined(OMR_OS_WINDOWS) */
```

Also there seems to already be an `#if defined(OMR_OS_WINDOWS)` on line 155. Could we just move this for loop after the check there?